packages/manager/modules/manager-components/src/edit-name/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+
+import '@ovh-ux/ui-kit';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+
+import component from './component';
+
+const moduleName = 'ovhManagerComponentsEditNameComponent';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    'ngTranslateAsyncLoader',
+  ])
+  .component('editServiceName', component)
```

Comment:
I think we have to name them like for others module:

```suggestion
const moduleName = 'ovhManagerComponentsEditName';

angular
  .module(moduleName, [
    'oui',
    'pascalprecht.translate',
    'ngTranslateAsyncLoader',
  ])
  .component('editServiceNameComponent', component)
```

Reply:
👍